android/build.gradle

```diff
@@ -45,7 +53,7 @@ repositories {
 }
 
 dependencies {
-    compile "com.facebook.react:react-native:+"  // From node_modules
+    compile "com.facebook.react:react-native:${_reactNativeVersion}"  // From node_modules
```

Comment:
React Native is not published to maven anymore, using `+` is the recommended method.